silhouette/Graphtec.py

```diff
@@ -842,6 +842,7 @@ def setup(self, media=132, speed=None, pressure=None, toolholder=None, pen=None,
       if pen:
         self.send_command(tool.sharpen_corners(0, 0))
       else:
+        # TODO: shouldn't be this also SU? why * 10 ?
```

Comment:
I think that times 10 was right (also I don't have a clue as to why they don't use SUs here), but I will have another look at it with wireshark and will report back on my findings.

Reply:
So I gave it another look and indeed the numbers are in 1/10th of millimeter, i.e. 0.1mm translates to 1, 0.5mm translates to 5, and so on.
To my surprise the FF$1,$2,$3 command behaves similar to the TJ command:
* $1 is updated immediately, as in I change the start ext. in SS (e.g. 0.1 to 0.5) and it will be transmitted accordingly in the next send command for both FF_,_,_ commands
* BUT $2 differs between the two FF_,_,_ commands: The first command repeats the old value (e.g. 0.1) and the second holds the new value (e.g. 0.5)

And as with the TJ command in the first send after reconnection of the Cameo it sends 0 as old value (probably because the old value is stored per connection or something). With this in mind, I think you might have been right about TJ being acceleration after all. I will investigate this further.

To sum things up times 10 here is correct, so I think you should alter that comment to state that 10 is correct to prevent future code reviewers from wondering about that and checking it again and again.